packages/arb-token-bridge-ui/tests/support/commands.ts

```diff
@@ -79,8 +79,8 @@ Cypress.Commands.add(
       // and returns a boolean. If it returns true, the recursion stops
       $input => $input.val() === text,
       {
-        log: false,
-        timeout: 180_000
+        log: true,
```

Comment:
don't need this anymore?

also do we need 240k timeout or can go back to 180k?